CHANGELOG.md

```diff
@@ -1,4 +1,6 @@
-## Unreleased
```

Comment:
We should keep the unreleased section. It useful for PRs that make changes but don't also cut a new version.